doc/configuration.md

```diff
@@ -1,415 +1,8 @@
-volume.block.mount\_options     | string    | block based driver (lvm)          | discard           | Mount options for block devices
-lvm.thinpool\_name              | string    | lvm driver                        | LXDPool           | Thin pool where images and containers are created.
-lvm.use\_thinpool               | bool      | lvm driver                        | true              | Whether the storage pool uses a thinpool for logical volumes.
-lvm.vg\_name                    | string    | lvm driver                        | name of the pool  | Name of the volume group to create.
-rsync.bwlimit                   | string    | -                                 | 0 (no limit)      | Specifies the upper limit to be placed on the socket I/O whenever rsync has to be used to transfer storage entities.
-volume.size                     | string    | appropriate driver                | 0                 | Default volume size
-volume.zfs.remove\_snapshots    | bool      | zfs driver                        | false             | Remove snapshots as needed
-volume.zfs.use\_refquota        | bool      | zfs driver                        | false             | Use refquota instead of quota for space.
-zfs.pool\_name                  | string    | zfs driver                        | name of the pool  | Name of the zpool
-zfs.clone\_copy                 | bool      | zfs driver                        | true              | Whether to use ZFS lightweight clones rather than full dataset copies.
-
-Storage pool configuration keys can be set using the lxc tool with:
-
-    lxc storage set [<remote>:]<pool> <key> <value>
-
-## Storage volume configuration
-
-Key                     | Type      | Condition                 | Default                               | Description
-:--                     | :--       | :--                       | :--                                   | :--
-size                    | string    | appropriate driver        | same as volume.size                   | Size of the storage volume
-block.filesystem        | string    | block based driver (lvm)  | same as volume.block.filesystem       | Filesystem of the storage volume
-block.mount\_options    | string    | block based driver (lvm)  | same as volume.block.mount\_options   | Mount options for block devices
-zfs.remove\_snapshots   | string    | zfs driver                | same as volume.zfs.remove\_snapshots  | Remove snapshots as needed
-zfs.use\_refquota       | string    | zfs driver                | same as volume.zfs.zfs\_requota       | Use refquota instead of quota for space.
-
-Storage volume configuration keys can be set using the lxc tool with:
-
-    lxc storage volume set [<remote>:]<pool> <volume> <key> <value>
+- [the LXD server itself](server.md)
+- [containers](containers.md) 
+- [networking](networks.md)
+- [profiles](profiles.md)
+- [storage pools and backends](storage.md)
```

Comment:
This is a nit but can we name this:
- `Server`
- `Containers`
- `Networking`
- `Profiles`
- `Storage`